Thai/Songs/ThaiSong_FanJa.md

```diff
@@ -34,3 +34,6 @@ by Torr  Nov 7, 2015
 Git/GitHub จงเจริญ โย่วๆ
 
 by [Off](http://www.facebook.com/off99555) อิอิ เทส ฮัลโหล ทดสอบ สตอรี่ บอทจงมา accept this pull request now
+
+==========================
+Try [canoeing](https://github.com/hackinteach/create-your-own-adventure/blob/master/Thai/travel.md)
```

Comment:
This link seems to be an absolute link to GitHub rather than a relative link.  If you are linking to another page in the create-your-own-adventure repository, please use relative links to do so.  See the README of this repository for more information.